mesonbuild/interpreter/interpreterobjects.py

```diff
@@ -511,6 +513,9 @@ def configtool_method(self, args: T.Tuple[str], kwargs: TYPE_kwargs) -> str:
     @noPosargs
     @typed_kwargs('dependency.partial_dependency', *_PARTIAL_DEP_KWARGS)
     def partial_dependency_method(self, args: T.List[TYPE_nvar], kwargs: 'kwargs.DependencyMethodPartialDependency') -> Dependency:
+        if kwargs['includes'] and not isinstance(self.held_object, dependencies.InternalDependency):
```

Comment:
Seems weird that this is the only thing in this commit. Perhaps you were going to do some squashing?